algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -447,7 +449,7 @@ case class DenseHLL(bits: Int, v: Bytes) extends HLL {
         count += 1
         res += 1.0
       } else {
-        res += java.lang.Math.pow(2.0, -mj)
+        res += HyperLogLog.powersOfNegativeTwo(mj)
```

Comment:
should this be negativePowersOfTwo, not powersOfNegativeTwo? the current name implies we are doing (-2)^(mj) which is not the same as (2)^(-mj)